components/cluster/cluster-glue/cluster-glue.p5m

```diff
@@ -0,0 +1,206 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2013 Adam Stevko. All rights reserved.
+# Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+<transform file path=usr/share/man/man1m/(.+).1m$ -> set action.hash usr/share/man/man8/%<1>.8 >
```

Comment:
Yes, the transformation ist still used.
I tried to change the makefiles to create the man pages in 1m but this produced too much errors.
It was easier to create the man files with names for man.8 but with the contents already changed to 1m and to rename the files afterwards with the transformation.